server/src/test/java/org/elasticsearch/cluster/coordination/AtomicRegisterCoordinatorTests.java

```diff
@@ -183,6 +183,17 @@ public void testCannotJoinClusterWithDifferentUUID() {
         // TODO: add more checks to avoid forming a mixed cluster between register based and traditional clusters
     }
 
+    @Override
+    @AwaitsFix(bugUrl = "ES-5645")
+    public void testClusterUUIDLogging() {
+        // Stateless masters do not lock in to a cluster UUID in a way that persists across restarts so this test doesn't make sense here
+    }
+
+    @Override
+    protected long extraStabilisationTime() {
+        return 300000; // TODO liveness improvements
```

Comment:
Is this still needed with the fix from https://github.com/elastic/elasticsearch/pull/94726?

Reply:
Possibly not. I expect we will  need to allow for _some_ extra time for stabilisation in these tests, especially if we make some of the possibly-async steps into genuinely-async ones. But this wasn't that, it's just the first thing I try when looking at these tests.